src/main/java/gregtech/common/metatileentities/MetaTileEntities.java

```diff
@@ -717,6 +717,7 @@ public static void init() {
          * - PCM's Ore Addon: 9500-9999
          * - GCM: 10000-10099
          * - MechTech: 10100-10499
+         * - MBT 10500 - 10999
```

Comment:
hmm i thought we are going to use 32k+ for mte ids for mbt considering thats where i reserved for ct materials

Reply:
its already reserved on the line directly below this one, this line is redundant

Reply:
lol
i cant read then

Reply:
This is the ID that MBT needs to extend other mods. (Botania Mana Hatach, Forge Energy Hatach, etc)